src/pages/Inventory.page.js

```diff
@@ -1,15 +1,24 @@
 import { BaseSwagLabPage } from './BaseSwagLab.page';
 
 export class InventoryPage extends BaseSwagLabPage {
+
     url = '/inventory.html';
 
     headerTitle = this.page.locator('.title');
 
     inventoryItems = this.page.locator('.inventory_item');
 
+    get inventoryItemName() { return this.page.locator('.inventory_item_name'); }
+
+    get inventoryPrice() { return this.page.locator('.inventory_item_price'); }   
+
     addItemToCartButton = this.page.locator('[id^="add-to-cart"]');
 
-    async addItemToCartById(id) {
+    sortDropdown = this.page.locator('.product_sort_container');     
```

Comment:
We have locators as class fields described there^^, then we have is as getter methods and here we go again with class fields. Its always better to stick to the one approach through whole project. So please, pick one approach (fileds OR getters) and stick to it in all page objects. Otherwise people who try to extend your code will try to find the difference between these locators and answer a question "why it differs?"